### PR TITLE
fix: fix bug

### PR DESCRIPTION
**Pull request by Han-Ya-Jun**
### Description
- fix release bug
- fix resource version display

Fixes # (issue)

### Checklist

- [x] 填写 PR 描述及相关 issue (write PR description and related issue)
- [x] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [x] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: src/dashboard/apigateway/apigateway/apis/web/resource_version/serializers.py [hunk 1]
```diff
@@ -129,7 +135,13 @@ def get_sdk_count(self, obj):
...
 
     def get_version(self, obj):
-        return obj.get("version")
+        version = obj.get("version")
+        if not version:
+            # 兼容历史数据
+            name = obj.get("name")
+            title = obj.get("title")
+            return f"{title}({name})"
+        return version
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apis/web/resource_version/serializers.py:144]: 这里直接`return obj.object_display`?

Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/apis/web/resource_version/serializers.py:144]: 这里取不到，因为前面传入的是queryset,这里的obj是一个dict

File: src/dashboard/apigateway/apigateway/apis/web/resource_version/serializers.py [hunk 2]
```diff
@@ -109,6 +112,9 @@ class ResourceVersionRetrieveOutputSLZ(serializers.Serializer):
     created_time = serializers.DateTimeField(help_text="创建时间")
     created_by = serializers.CharField(help_text="创建人")
 
+    def get_version(self, obj: ResourceVersion):
```
Comment (zhu327) [inline src/dashboard/apigateway/apigateway/apis/web/resource_version/serializers.py:115]: 有一种source的写法可以省掉这个方法

Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/apis/web/resource_version/serializers.py]: done

**Review by wklken — APPROVED** [review]

**Review by zhu327 — APPROVED** [review]

**Pull request merged**